MapboxNavigationUI/NavigationMapView.swift

```diff
@@ -35,16 +32,16 @@ open class NavigationMapView: MGLMapView {
             style.removeSource(source)
         }
         
-        let polyline = shape(for: route)
+        let polyline = shape(describing: route)
         
         if let source = style.source(withIdentifier: sourceIdentifier) as? MGLShapeSource {
             source.shape = polyline
         } else {
-            let geoJSONSource = MGLShapeSource(identifier: sourceIdentifier, shape: polyline, options: nil)
-            style.addSource(geoJSONSource)
+            let lineSource = MGLShapeSource(identifier: sourceIdentifier, shape: polyline, options: nil)
+            style.addSource(lineSource)
             
-            let line = lineStyle(MGLLineStyleLayer(identifier: routeLayerIdentifier, source: geoJSONSource))
-            let lineCasing = lineCasingStyle(MGLLineStyleLayer(identifier: routeLayerCasingIdentifier, source: geoJSONSource))
+            let line = routeStyleLayer(identifier: routeLayerIdentifier, source: lineSource)
```

Comment:
Heh, I completely forgot that we already have a NavigationMapViewDelegate protocol that would make a nice home for these methods.